staging/src/k8s.io/client-go/rest/request_test.go

```diff
@@ -2065,6 +2065,13 @@ func testRESTClientWithConfig(t testing.TB, srv *httptest.Server, contentConfig
 	if err != nil {
 		t.Fatalf("failed to create a client: %v", err)
 	}
+
+	// the tests by default can continue to use NoBackoff
+	// to maintain shorter test duration.
```

Comment:
is there so much difference? are we retrying on the tests?

if possible I think that is preferable to use same defaults in the tests

Reply:
the fact that we had to adjust the tests to keep passing with the default changing to backoff is a strong signal that this will be a user-visible change that will disrupt/break some consumers